blender/arm/logicnode/math/LN_math.py

```diff
@@ -63,6 +63,9 @@ def set_enum(self, value):
                 while (len(self.inputs) > 1):
                     self.inputs.remove(self.inputs.values()[-1])
         self['property0'] = value
+        if (self.property0 == 'Round'):
+            self.inputs[1].name = 'Precision'
+        else: self.inputs[1].name = 'Value 1'
```

Comment:
Hi, this will fail if you switch from "Round" to some operation that has only one input, e.g. "Sine". Better change the `else` to `elif len(self.inputs) > 1`. Did not see this while we were talking about it on Discord, sorry.

Reply:
you are right i did not see the error in the console log... i just see it working as expected when i tested all the cases,

thanks im gonna make the change